src/ansi-c/ansi_c_internal_additions.cpp

```diff
@@ -131,6 +131,15 @@ void ansi_c_internal_additions(std::string &code)
     // NOLINTNEXTLINE(whitespace/line_length)
     "__CPROVER_bool __CPROVER_threads_exited[__CPROVER_constant_infinity_uint];\n"
     "unsigned long __CPROVER_next_thread_id=0;\n"
+
+    // traces
+    "void CBMC_trace(int lvl, const char *event, ...);\n"
+
+    // pointers
+    "unsigned __CPROVER_POINTER_OBJECT(const void *p);\n"
```

Comment:
Convention: I think these are usually `__CPROVER_lowercase_name`

Reply:
The above seems to have been re-introduced after it got moved out into cprover_builtin_headers.h